docker/test/util/process_functional_tests_result.py

```diff
@@ -59,8 +59,12 @@ def process_test_log(log_path, broken_tests):
 
                 total += 1
                 if TIMEOUT_SIGN in line:
-                    failed += 1
-                    test_results.append((test_name, "Timeout", test_time, []))
+                    if test_name in broken_tests:
+                        success += 1
+                        test_results.append((test_name, "OK", test_time, []))
```

Comment:
Is it possible to write smth like "KNOWN_BROKEN" instead of "OK" here?

Reply:
Status can be wrong if we calculate the number of `OK`, isn't it?

Reply:
It rather affects the color of the report. Anything unknown is written as yellow color

Reply:
https://github.com/ClickHouse/ClickHouse/blob/master/tests/ci/report.py#L283